libzfs.pyx

```diff
@@ -1013,6 +1015,11 @@ cdef class ZFS(object):
                 'dataset': name.split('@')[0],
                 'id': name
             })
+            if simple_createtxg:
+                create_txg = libzfs.zfs_prop_get_int(handle, zfs.ZFS_PROP_CREATETXG)
+                snap_data.update({
+                    'createtxg': str(create_txg)
+                })
```

Comment:
Since this is already populated dmustats, we basically get for free. Maybe we should always get it like for `name` above, and populate in above snap_data.update()